fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -45,6 +45,15 @@ def self.run(params)
             version = latest_candidate_build.train_version
             build_number = latest_candidate_build.build_version
           end
+        elsif version == 'live'
+          UI.message("Looking for live version...")
+          latest_live_version = app.live_version(platform: platform)
+
+          UI.user_error!("Could not find live version for your app, please try setting 'latest' or a specific version") if latest_live_version.version.nil?
```

Comment:
Should this be `if latest_live_version.nil?` I _think_ if there is no live version then `latest_live_version` will be nil 🤔

```suggestion
          UI.user_error!("Could not find live version for your app, please try setting 'latest' or a specific version") if latest_live_version.nil?
```

Reply:
ah true, good catch thx 😅